packages/node/tsconfig.json

```diff
@@ -1,15 +1,12 @@
 {
   "compilerOptions": {
-    "strict": true,
-    "esModuleInterop": true,
     "lib": ["ES2021", "DOM"],
-    "target": "ES2021",
     "module": "nodenext",
-    "outDir": "dist",
-    "sourceMap": false,
-    "declaration": false,
-    "skipLibCheck": true,
-    "moduleResolution": "nodenext"
+    "moduleResolution": "nodenext",
```

Comment:
I prefer `node16` over `nodenext` as its clearer that it won't magically change in the future

Reply:
I dunno what exactly the differences are between `node16` and `nodenext`. Given I didn't change the value, I'm leaning more towards not changing it in this PR. I'd really like to not break anything, though I doubt it would break anything. @styfle what do you think?

Reply:
fair enough. we can make another pass on this in the future